src/components/GuestNamePicker.vue

```diff
@@ -7,7 +7,9 @@
 		:out-transition="true"
 		size="small"
 		:show.sync="show">
-		<div class="modal__content" data-cy="guestNameModal">
+		<form class="modal__content"
```

Comment:
Not needed, just migate this to `NcDialog` and set `is-form` prop :)

Reply:
That sounds more reasonable, I will take a look :D